README.md

```diff
@@ -4,6 +4,19 @@ Various PSR3-compatible logging adapters.
 
 ## Adapter-specific notes
 
+### Psr3ConsoleLogger
+- Since this is ambiguous regarding whether logged messages are affected by
+  PHP's output buffering, we recommend using either `Psr3EchoLogger` or
+  `Psr3StdOutLogger` instead.
```

Comment:
Probably should state it extends the StdOut one.

Reply:
What does that gain us?

I had it extend in order to avoid backwards-compatibility breaking changes, but I don't want people to use this `Psr3ConsoleLogger` in the future. I thought about officially deprecating it, but thought it better to simply provide better options for now.

Reply:
It's best to state current behaviour, state the recommendation, and warn of impending deprecation. What do we gain? Clarity of state and direction?